storybookjs/storybook
# [app:angular] Fixed dependency injection for components

Issue:
DI on dynamic components doesn't work 

## What I did
Fixed dependency injection on dynamic components 

## How to test
inject dependency into component, check that dependency is defined

Is this testable with jest or storyshots?
no

Does this need a new example in the kitchen sink apps?
no

Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.
